client/state/data-layer/wpcom/read/sites/notification-subscriptions/delete/index.js

```diff
@@ -0,0 +1,60 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { READER_UNSUBSCRIBE_TO_NEW_POST_NOTIFICATIONS } from 'state/action-types';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { dispatchRequestEx } from 'state/data-layer/wpcom-http/utils';
+import { errorNotice } from 'state/notices/actions';
+import { translate } from 'i18n-calypso';
+import { bypassDataLayer } from 'state/data-layer/utils';
+import { subscribeToNewPostNotifications } from 'state/reader/follows/actions';
+
+export function fromApi( response ) {
+	const isAdded = !! ( response && response.success );
+	if ( ! isAdded ) {
+		throw new Error(
+			`Unsubscription from new post notifications failed with response: ${ JSON.stringify(
+				response
+			) }`
+		);
+	}
+
+	return response;
+}
+
+export function requestNotificationUnsubscription( action ) {
+	return http(
+		{
+			path: `/read/sites/${ action.payload.blogId }/notification-subscriptions/delete`,
+			method: 'POST',
+			apiNamespace: 'wpcom/v2',
+		},
+		action
+	);
+}
+
+export function receiveNotificationUnsubscriptionError( action ) {
+	return [
+		errorNotice(
+			translate( 'Sorry, we had a problem unsubscribing you from notifications. Please try again.' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Sorry, we had a problem unsubscribing. Please try again.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
do we have access to the site name so we can be more descriptive and let them know which site failed?

Reply:
It's not quite as easy to grab it now, because we don't have easy access to `getState()`. Previously we could just have used the `getSiteTitle` selector.

I'll ponder how best to do it and add in a later PR.